packages/devtools_app/test/vm_developer/object_inspector/vm_code_display_test.dart

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math';
+
+import 'package:devtools_app/devtools_app.dart';
+import 'package:devtools_app/src/screens/vm_developer/vm_code_display.dart';
+import 'package:devtools_app/src/screens/vm_developer/vm_service_private_extensions.dart';
+import 'package:devtools_app/src/shared/table.dart';
+import 'package:devtools_test/devtools_test.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:vm_service/vm_service.dart';
+
+void main() {
+  late MockCodeObject mockCodeObject;
+
+  const windowSize = Size(4000.0, 4000.0);
+
+  group('VmCodeDisplay', () {
+    setUpAll(() {
+      setGlobal(IdeTheme, IdeTheme());
+
+      mockCodeObject = MockCodeObject();
+      final testCode = Code(
```

Comment:
Should testCode be in vm_developer_test_utils.dart?

Reply:
That's a good question. Maybe once more functionality is written, but I think it's worth leaving here for now.